pkg/plugin/utils.go

```diff
@@ -90,6 +90,8 @@ func makeEmptyConcreteTypeArray(val interface{}) (interface{}, error) {
 		return []float64{}, nil
 	case bool:
 		return []bool{}, nil
+	case time.Time:
+		return []time.Time{}, nil
```

Comment:
This is really key.  As you populate fields in a data frame, Grafana's data layer infers the field type from this array type.